src/vs/workbench/api/common/extHostTypeConverters.ts

```diff
@@ -2907,6 +2908,24 @@ export namespace TerminalQuickFix {
 	}
 }
 
+export namespace TerminalCompletionList {
+	export function from(completionList: vscode.TerminalCompletionList, converter: Command.ICommandsConverter, disposables: DisposableStore): extHostProtocol.TerminalCompletionListDto {
+		return {
+			...completionList,
+			resourceRequestConfig: completionList.resourceRequestConfig ? TerminalResourceRequestConfig.from(completionList.resourceRequestConfig, converter, disposables) : undefined,
+		};
+	}
+}
+
+export namespace TerminalResourceRequestConfig {
```

Comment:
do I need to pass in converter and disposables here? they aren't used

Reply:
No need if they're not used